clients/fides-js/src/components/fides.css

```diff
@@ -1179,6 +1181,17 @@ div#fides-overlay-wrapper .fides-toggle .fides-toggle-display {
   );
 }
 
+@media screen and (max-width: 768px) {
+  .fides-banner-button-group.fides-button-group-i18n {
+    min-height: 68px;
+  }
+  .fides-i18n-menu {
+    position: absolute;
+    left: var(--fides-overlay-padding);
+    bottom: var(--fides-overlay-padding);
+  }
+}
+
```

Comment:
This mimics what we're doing on the modal to align language and privacy on the same line